state_management/csharp/sdk/README.md

```diff
@@ -14,19 +14,6 @@ This section shows how to run applications at once using [multi-app run template
 
 1. Open a new terminal window and run  `order-processor` using the multi app run template defined in [dapr.yaml](./dapr.yaml):
 
-<!-- STEP
-name: Install Dotnet dependencies
--->
-
-    ```bash
-    dotnet restore
```

Comment:
The test was failing because the `dotnet build` was not happening in order-processor subfolder.  You could either set the working directory or cd into it.

However your simplification of restore and build made me think we should just further simplify.  Dotnet language tools do a restore and build on run.  So let's go straight to the run, which is done as a part of `dapr run -f .`.